synfig-studio/src/gui/canvasview.cpp

```diff
@@ -577,22 +577,20 @@ CanvasView::CanvasView(etl::loose_handle<Instance> instance,etl::handle<CanvasIn
 
 	//info("Canvasview: Before big chunk of allocation and tabling stuff");
 	//create all allocated stuff for this canvas
-	Gtk::Fixed *widget_space = Gtk::manage(new Gtk::Fixed);
-	widget_space->set_size_request(4,4);
-	widget_space->show();
 
 	Gtk::Widget *widget_work_area = create_work_area();
+	widget_work_area->set_margin_top(4);
 	init_menus();
 	Gtk::Widget *widget_top_bar = create_top_toolbar();
 	Gtk::Widget *widget_stopbutton = create_stop_button();
 	Gtk::Widget *widget_right_bar = create_right_toolbar();
+	widget_right_bar->set_margin_start(4);
```

Comment:
Figured this isn't really needed, just the work area widget will do because `widget_space` was added horizontally whereas the right toolbar is a vertical widget and there's no ruler to the right. So no extra spaces needed at all...

Reply:
Got it ! made the required changes in the latest commit :)